bridge/framework/base.py

```diff
@@ -40,15 +46,25 @@ def start_local_services(self):
         client = docker.from_env()
         if self.enable_postgres:
             self.start_local_postgres(client)
+        if self.enable_redis:
+            self.start_local_redis(client)
 
     def start_local_postgres(self, client: docker.DockerClient) -> None:
         service = PostgresService(client=client)
         service.start()
 
+    def start_local_redis(self, client: docker.DockerClient) -> None:
+        config = RedisConfig()
+        service = RedisService(client=client, config=config)
+        service.start()
+        self.configure_redis(config)
```

Comment:
I think this configuration call should happen in configure_services() instead to match the new pattern with postgres.